qa/L0_backend_python/python_unittest.py

```diff
@@ -30,17 +30,17 @@
 
 import test_util as tu
 import unittest
-import tritonclient.http as httpclient
```

Comment:
Why move from http to grpc?

Reply:
It didn't support the `client_timeout` parameter in `infer`.